src/Modules/Profile/Commands.Profile/InvokePowerBIRestMethod.cs

```diff
@@ -115,16 +121,16 @@ public override void ExecuteCmdlet()
                             response = await client.GetAsync(url);
                             break;
                         case PowerBIWebRequestMethod.Post:
-                            response = await client.PostAsync(url, new StringContent(body));
+                            response = await client.PostAsync(url, new StringContent(body, Encoding.UTF8, this.ContentType));
```

Comment:
This will probably never change, but you could declare the encoding in a variable.

Reply:
It looks like our APIs expect UTF8 which is why I didn't allow it to be configured